lua/pixelui/core/cl_imgur.lua

```diff
@@ -18,17 +18,54 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 local materials = {}
 
+local imgurQueue = {}
+
+function imgurQueue.new()
+    return { coroutines = {} }
```

Comment:
Keep styling consistency
```suggestion
    return {coroutines = {}}
```